src/index.js

```diff
@@ -61,6 +64,9 @@ class Mongo {
       },
       convertUserIdToMongoObjectId: true,
       caseSensitiveUserName: true,
+      convertSessionIdToMongoObjectId: true,
+      idProvider: null,
+      dateProvider: (() => Date.now()),
```

Comment:
With respect to the comment above, default dateProvider should probably be:
`(date?: Date) => date ? date.getTime() : Date.now()`
And there is no need for the wrapping parenthesis here.